packages/lib/src/vis/heatmap/HeatmapVis.tsx

```diff
@@ -127,12 +122,12 @@ function HeatmapVis(props: Props) {
           }}
         />
         <HeatmapMesh
-          values={toTypedNdArray(dataArray, Float32Array)}
+          values={toTextureSafeNdArray(dataArray)}
```

Comment:
This is still where the conversion to `Float32Array` takes place, but now, in addition to converting `number[]`, it also converts other typed arrays.